build.gradle

```diff
@@ -18,9 +18,6 @@ allprojects {
     repositories {
         google()
         mavenCentral()
-        maven {
-            url "https://s01.oss.sonatype.org/content/repositories/snapshots/"
-        }
```

Comment:
Let's leave this here in case anyone wants to test it out with a snapshot version of the SDK

Reply:
Understood now